primitives/avn-common/src/event_types.rs

```diff
@@ -698,13 +739,15 @@ pub enum EventData {
     LogNftEndBatchListing(NftEndBatchListingData),
     LogAvtGrowthLifted(AvtGrowthLiftedData),
     LogLowerClaimed(AvtLowerClaimedData),
-    LogLiftedToPredictionMarket(LiftedToPredictionMarketData),
+    LogLiftedToPredictionMarket(LiftedData),
+    LogErc20Transfer(LiftedData),
 }
 
 impl EventData {
     #[allow(unreachable_patterns)]
     pub fn is_valid(&self) -> bool {
         return match self {
+            // LogLowerClaimed missing. TODO add and remove unreachable patterns.
```

Comment:
I am assuming we never noticed because lower uses the new flow (has never worked with EthereumEvents)?

Reply:
yes, exactly.